app/views/projects/samples/transfers/_transfer_modal.html.erb

```diff
@@ -19,6 +19,7 @@
         <div class="flex flex-row-reverse">
           <%= form.submit t("projects.samples.transfers._transfer_modal.submit_button"),
                       class: "button button--size-default button--state-primary",
+                      disabled: !@projects.count.positive?,
```

Comment:
**nitpick (blocking):** Can you change this to `@projects.count.zero?`

Reply:
Sure thing. updated in 9fa93bc4280c6f9ae701c65f36d8b5d76e577485